lang/cs/Org.Apache.REEF.Client/Local/LocalClient.cs

```diff
@@ -135,10 +140,22 @@ public IDriverHttpEndpoint SubmitAndGetDriverUrl(IJobSubmission jobSubmission)
             HttpClientHelper helper = new HttpClientHelper();
             _driverUrl = helper.GetDriverUrlForLocalRuntime(fileName);
 
+            _jobStatus = ApplicationState.SUBMITTED;
             Logger.Log(Level.Info, "Submitted the Driver for execution. Returned driverUrl is: " + _driverUrl);
             return helper;
         }
 
+        /// <summary>
+        /// Return current Job status
+        /// </summary>
+        /// <returns></returns>
+        [Unstable("0.14", "Working in progress for rest API status returned")]
+        public async Task<ApplicationState> GetJobStatus()
+        {
+            await Task.Delay(1000); //1 seconds delay
```

Comment:
We don't really support status for local run time yet. This s a fake one. The API is defined in the interface, YarnREEFClient has a proper implementation because it can get status from Yarn Rest API. For local, we need to come out a proper implementation. I will create a Jira to follow it up.